packages/cli/bin/cli.js

```diff
@@ -37,6 +37,10 @@ env.register(
   path.join(__dirname, '../generators/controller'),
   'loopback4:controller'
 );
+env.register(
```

Comment:
Out of scope for the PR, but we probably should refactor the list of generators into a separate file, such as:

generators.js
```js
module.exports =  {
  app: path.join(__dirname, '../generators/app'), // Path
  controller: path.join(__dirname, '../generators/controller'), // Path
  another: AnotherGenerator, // Generator class
}
```

Reply:
Ideally, I'd like us to discover all generators automatically by scanning `generators` directory. I believe Yeoman is already doing that (or was doing that in earlier versions) for projects that are following prescribed project layout (see our generator-loopback for an example).

Reply:
I'm not sure if we really want to use Yeoman discovery, which is one of areas that caused confusions in the past.

Reply:
I agree we probably don't want to use Yeoman discovery, I'd like us to write our own to follow the conventions we like most. But yeah, it's out of scope of this pull request.